.github/workflows/lint-autoformat.yml

```diff
@@ -0,0 +1,64 @@
+name: Apply lint suggestions
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+    branches: [main]
+
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [gh/**/base]
+
+jobs:
+  # NB: This doesn't work with forked PR https://github.com/parkerbxyz/suggest-changes/issues/33 :(
+  checkout-pytorch-grab-lintrunner:
+    permissions:
+      contents: read
+      pull-requests: read
+    runs-on: lf.linux.2xlarge
+    if: ${{ github.repository_owner == 'pytorch' }}
+    outputs:
+      lintrunner-script: ${{ steps.store-lintrunner-script.outputs.lintrunner-script }}
+    steps:
+      - name: Checkout PyTorch
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
```

Comment:
Maybe put a note here to point out what @kit1980 mentions that this GHA checkouts the PR head when it's a `pull_request_target`